src/app/pages/home/prysmatic-home.page.component.html

```diff
@@ -0,0 +1,371 @@
+                        <a class="is-link" target="_blank" href="https://github.com/prysmaticlabs/prysm/blob/master/CONTRIBUTING.md">here</a>
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section bg-grey-lightest section-background is-size-5 shadow-lg">
+    <div class="container ">
+        <div class="columns">
+            <div class="column">
+                <h1 class="title">
+                    <b class="text-teal">Prysm</b> Specifications</h1>
+                <table class="table w-full bg-transparent prysm-specifcations-table">
+                    <tbody>
+                        <tr>
+                            <th>Network</th>
+                            <td>
+                                <p>Ethereum
+                                    <fa-icon [icon]="faEthereum"></fa-icon> Serenity, Shasper</p>
+                                <p>(Sharding + Casper Proof of Stake)</p>
+                            </td>
+                        </tr>
+                        <tr>
+                            <th>Implementation Language</th>
+                            <td>Golang</td>
+                        </tr>
+                        <tr>
+                            <th>networking protocol</th>
+                            <td>Libp2p from Protocol Labs</td>
+                        </tr>
+                        <tr>
+                            <th>Interprocess Communication</th>
+                            <td>Google's gRPC framework</td>
+                        </tr>
+                        <tr>
+                            <th>Test and Build Pipeline</th>
+                            <td>Google's Bazel build tool</td>
+                        </tr>
+                        <tr>
+                            <th>Database</th>
+                            <td>BoltDB, a write-optimized, persistent key-value store</td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div class="column">
+                <div class="w-full">
+                    <figure class="image is-square">
+                        <img src="./assets/product/prysm.gif" class="rounded shadow-lg" alt="product prysm">
+                    </figure>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section bg-grey-light ">
+    <div class="container bg-grey-light ">
+        <h1 class="title">Sponsors and Grants</h1>
+        <h2 class="subtitle">
+            Thank you to all who have contributed to Prysmatic Labs; we are eternally grateful.
+        </h2>
+
+        <div class="columns m-auto is-multiline is-mobile">
+            <div class="column text-center is-half-mobile ">
+                <a href="https://www.aragon.one" alt="argon-one">
+                    <img src="./assets/sponsors/aragonnest.svg" alt="aragonnest" width="100px" />
```

Comment:
Please do not use inline styles like "width". This is trivially done via CSS selector 
```css
.column a img {
  width: 100px;
}
```